File: go/chapter7/deli.go
package main

import "fmt"

func main() {
	sandwichOrders := []string{"bread", "apple", "vanilla", "brown", "tuna"}
	finishedSandwiches := make([]string, 0)

	for _, v := range sandwichOrders {
		fmt.Printf("I made your %s sandwich\n", v)
		finishedSandwiches = append(finishedSandwiches, v)
	}

	fmt.Println(finishedSandwiches)
}

File: javascript/chapter7/deli.js
const sandwichOrders = ["bread", "apple", "vanilla", "brown", "tuna"];
const finishedSandwiches = [];

for (const value of sandwichOrders) {
	console.log(`I made your ${value} sandwich`);
	finishedSandwiches.push(value);
}

console.log(finishedSandwiches);
